### DISCUSSION TITLE
ChatOps is not running in stackstom

### DISCUSSION THREAD

**Discussion by anujku3x** [StackStorm/st2#6249] [ChatOps]
![image](https://github.com/user-attachments/assets/03ce9ce5-98ca-4aab-9325-94a0cf60e8c4)
Hi Team, Please help me with ChatOps integration. As per the documentation all the configuration and setup we did, ChatOps is not running in StackStorm even after restarting the server. I set up stackstorm log back ago now we need to integrate ChatOps.

**Comment by fdrab** [comment]
none of the services are running in your screenshot
